package.json

```diff
@@ -25,22 +25,23 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.6.0",
-    "@actions/http-client": "^2.0.1",
+    "@actions/core": "^1.10.0",
+    "@actions/http-client": "^2.1.0",
     "form-data": "^4.0.0"
   },
   "devDependencies": {
-    "@types/node": "^18.7.8",
-    "@typescript-eslint/eslint-plugin": "^5.33.1",
-    "@typescript-eslint/parser": "^5.33.1",
-    "@vercel/ncc": "^0.34.0",
-    "eslint": "^7.32.0",
-    "eslint-plugin-github": "^4.3.2",
-    "eslint-plugin-jest": "^25.3.2",
-    "jest": "^27.2.5",
+    "@types/node": "^18.14.6",
```

Comment:
this should probably be 20 since it's LTS and the action uses 20 since https://github.com/tsickert/discord-webhook/pull/164